packages/dev/core/src/Physics/v2/physicsBody.ts

```diff
@@ -335,6 +335,16 @@ export class PhysicsBody {
         return this._physicsPlugin.unregisterOnBodyCollide(this, func);
     }
 
+    /**
+     * Enable or disable collision callback for this PhysicsBody.
+     * `registerOnCollide` method will enable collision callback and `unregisterOnCollide` will disable them.
+     * Registering a collision callback on the plugin and enabling collision per body is faster than
+     * registering callback per PhysicsBody.

```

Comment:
missing @param enabled, cc @RaananW just in case it should be activated here ?

Reply:
Fixed.